drake/CMakeLists.txt

```diff
@@ -19,9 +19,9 @@ list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
 
 if(WITH_PYTHON_3)
   set(Python_ADDITIONAL_VERSIONS 3.5)
-  find_package(PythonInterp 3 REQUIRED)
+  find_package(PythonInterp 3)
```

Comment:
Specifying the python version names explicitly is an unfortunate hack in pods.cmake, and will break with every new python version number (unless we just guess a bunch of new future version numbers). Can we wait to see if this is actually a problem, since it's not causing issues on any of our test servers?